app/gui/view/graph-editor/src/new_node_position.rs

```diff
@@ -29,6 +29,12 @@ pub mod free_place_finder;
 /// === New Node Positioning ===
 /// ============================
 
+/// The maximum distance to the nearest grid line in Magnet Alignment algorithm.
```

Comment:
Distance is in pixels, isn't it? Add the info to the docs (or to the constant name, e.g. in suffix `_PX`)

Reply:
No, the distance is in "scene units" (as coordinates). I mentioned it in the docs.